### ISSUE TITLE
TypeError: Path must be a string. Received undefined

### ISSUE THREAD

**Issue by redheadedstep** [smallhelm/less-to-js#1]
Clean-CSS 3.2.9 dependency in package.json has a bug that was fixed in later revisions with Node 6+.

Running 

```
Marks-iMac:react-es hobby$ ./node_modules/.bin/less-to-js widgets/**/*.less widgets/**/*.css
path.js:7
    throw new TypeError('Path must be a string. Received ' + inspect(path));
    ^

TypeError: Path must be a string. Received undefined
    at assertPath (path.js:7:11)
    at Object.dirname (path.js:1324:5)
    at new CleanCSS (/Users/hobby/Projects/react-es/node_modules/clean-css/lib/clean.js:55:130)
    at minifyCSS (/Users/hobby/Projects/react-es/node_modules/less-to-js/index.js:7:10)
    at /Users/hobby/Projects/react-es/node_modules/less-to-js/index.js:20:42
    at /Users/hobby/Projects/react-es/node_modules/less/lib/less/render.js:35:17
    at /Users/hobby/Projects/react-es/node_modules/less/lib/less/parse.js:63:17
    at ImportVisitor.finish [as _finish] (/Users/hobby/Projects/react-es/node_modules/less/lib/less/parser/parser.js:183:28)
    at ImportVisitor._onSequencerEmpty (/Users/hobby/Projects/react-es/node_modules/less/lib/less/visitors/import-visitor.js:35:14)
    at ImportSequencer.tryRun (/Users/hobby/Projects/react-es/node_modules/less/lib/less/visitors/import-sequencer.js:50:14)
```

This is fixed in later versions of clean-css.

If you can change `package.json` from

```
"dependencies": {
    "clean-css": "3.2.9",
    "css-to-css.js": "^1.0.0",
    "each-async": "1.1.1",
    "globby": "1.2.0",
    "less": "2.5.0"
  }
```

to

```
"dependencies": {
    "clean-css": "^3.2.9",
    "css-to-css.js": "^1.0.0",
    "each-async": "1.1.1",
    "globby": "1.2.0",
    "less": "2.5.0"
  }
```

or update clean-css to 3.4.25, that would fix this issue.

**Comment by farskipper** [comment]
alright, just released `v1.1.0` that has your dependancy change in it

**Comment by farskipper** [comment]
I also just released `v2.0.0` which updates all dependencies to their latest
